extensions/ql-vscode/tsconfig.json

```diff
@@ -1,3 +1,32 @@
 {
-  "extends": "./node_modules/typescript-config/extension.tsconfig.json"
+  "$schema": "http://json.schemastore.org/tsconfig",
```

Comment:
Is this line important?

Reply:
I don't presently know of anything that requires it but it occurred in other tsconfig files in the previous state of this repo, and it seemed like fine documentation. I have no overwhelmingly strong opinion about including or eliminating it.

Reply:
I'm fine with keeping it. I've just never seen it explicitly before. I think vscode adds it automatically, but other tools may not.